x-pack/solutions/search/plugins/search_playground/public/hooks/use_load_fields_by_indices.ts

```diff
@@ -35,13 +35,16 @@ export const useLoadFieldsByIndices = ({
 }: Pick<UseFormReturn<ChatForm>, 'watch' | 'getValues' | 'setValue'>) => {
   const usageTracker = useUsageTracker();
   const selectedIndices = watch(ChatFormFields.indices);
-  const { fields } = useIndicesFields(selectedIndices);
+  const { fields, isFetched } = useIndicesFields(selectedIndices);
 
   useEffect(() => {
-    const [queryFields, sourceFields] = getValues([
```

Comment:
`getValues([])` doesn't type the return value correctly. switching to `getValues()` and destructing ensured the form fields we are using have the correct type instead of `any` or `never`.